### ISSUE TITLE
mdpow-ghyd fails with NameError

### ISSUE THREAD

**Issue by orbeckst** [Becksteinlab/MDPOW#10]
Running `mdpow-ghy DIR` fails with
```
Traceback (most recent call last):
  File "/nfs/homes2/oliver/.local/bin/mdpow-ghyd", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/nfs/homes2/oliver/Projects/Methods/MDPOW/scripts/mdpow-ghyd", line 224, in <module>
    force=opts.force, stride=opts.stride, permissive=opts.permissive)
  File "/nfs/homes2/oliver/Projects/Methods/MDPOW/scripts/mdpow-ghyd", line 115, in run_ghyd
    gwat.analyze(**akw)
  File "/nfs/homes2/oliver/Projects/Methods/MDPOW/mdpow/fep.py", line 931, in analyze
    DY = numpy.array([x.error[0]  for x in xvgs])   # takes a while: computes correl.time
  File "build/bdist.linux-x86_64/egg/gromacs/fileformats/xvg.py", line 457, in error
  File "build/bdist.linux-x86_64/egg/gromacs/fileformats/xvg.py", line 436, in _correlprop
  File "build/bdist.linux-x86_64/egg/gromacs/fileformats/xvg.py", line 397, in _tcorrel
  File "build/bdist.linux-x86_64/egg/gromacs/analysis/__init__.py", line 77, in <module>
  File "build/bdist.linux-x86_64/egg/gromacs/analysis/plugins/__init__.py", line 147, in <module>
NameError: name 'p' is not defined
```

This seems to be an issue with GromacsWrapper (I am running the latest develop).

**Comment by orbeckst** [comment]
@ianmkenney did you notice this problem?

**Comment by ianmkenney** [comment]
I haven't seen this problem. I normally calculate the hydration free energy in an ipython session

**Comment by orbeckst** [comment]
On 18 Jan, 2016, at 00:20, Ian Kenney wrote:

> I haven't seen this problem. I normally calculate the hydration free energy in an ipython session

Interactive is not useful for the workflow. We need a script that dumps the results into a file as shown in the mdpow-ghyd script. See #11 and #12.

**Comment by orbeckst** [comment]
The problem was due to a bug in the development version of GromacsWrapper and was just fixed in Becksteinlab/GromacsWrapper@c48f4d7eaa08688abae92655b19d15edeb86b79b
